### PR TITLE
Fix: Changelog

### PR DESCRIPTION
**Pull request by kwek20**
Changelog for https://github.com/iotaledger/iota-sdk/pull/982

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

File: sdk/CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.3 - 2023-MM-DD
+
+### Added
+
+- `last_sync` field to `ClientInner` instead of a local static ref;
+
```
Comment (Thoralf-M) [inline sdk/CHANGELOG.md:27]: Or maybe actually also add `- Clients returning the default protocol parameters when multiple Client instances are used;` in a fixed section?

File: sdk/CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.3 - 2023-MM-DD
+
+### Added
+
+- `last_sync` field to `ClientInner` instead of a local static ref;
```
Comment (thibault-martinez) [inline sdk/CHANGELOG.md]: I think I'd rather have a `Fixed` as it's actually fixing an issue.

File: sdk/CHANGELOG.md (hunk 1 above)
Comment (thibault-martinez) [reply sdk/CHANGELOG.md:27]: Yeah, that

Comment (thibault-martinez) [reply sdk/CHANGELOG.md:27]: Just do as Philipp did I guess https://github.com/iotaledger/iota-sdk/pull/982/files#diff-9241e7acbf9a7e53ca50142ef108f7a4edda10741e3995966bf4e66b5da7a66dR26

File: sdk/CHANGELOG.md (hunk 2 above)
Comment (kwek20) [reply sdk/CHANGELOG.md]: Ive left this as is and added thoralfs suggestion to fixed, so we got both 🎉

Comment (thibault-martinez) [reply sdk/CHANGELOG.md]: Yeah I mean, the field is not public, shouldn't really be in a changelog

**Review by thibault-martinez — APPROVED** [review]

**Review by qrayven — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**